src/config/config.ts

```diff
@@ -14,6 +14,13 @@ const defaultConfig: ZKitConfig = {
     skipFiles: [],
     c: false,
     json: false,
+    simplification: {
+      o0: false,
+      o1: false,
+      o2: false,
```

Comment:
I think these flags can't be applied together. So better do something like this:

```
optimization: "o1"
```